frontend/packages/kubevirt-plugin/src/components/virtualization-overview/HighPriorityAlerts.tsx

```diff
@@ -50,21 +47,22 @@ const AlertTitle = ({ alert }) => {
 };
 
 const HighPriorityAlerts: React.FC = () => {
-  const filteredAlerts: ConsoleAlert[] = useFilteredAlerts(isKubeVirtHighPriorityAlert);
+  const [filteredAlerts, alertsLoaded] = useFilteredAlerts(isKubeVirtHighPriorityAlert);
 
   return (
     <div className="kv-high-priority-alerts">
-      {filteredAlerts.map((alert) => {
-        const severity = alert?.labels?.severity as AlertSeverity;
-        return (
-          <Alert
-            isInline
-            variant={asAlertVariant(severity)}
-            title={<AlertTitle alert={alert} />}
-            className="kv-high-priority-alerts__alert"
-          />
-        );
-      })}
+      {alertsLoaded &&
```

Comment:
Nit: when there's nothing to render, you can consider returning `null` instead of rendering an effectively empty container `div` element:

```tsx
if (!alertsLoaded) {
  return null;
}

return (
  <div className="kv-high-priority-alerts">
    {/* render filteredAlerts here */}
  </div>
);
```

I'm OK with the current code, this is just a suggestion for the future.

Reply:
Good point, thanks!

Reply:
Fixed!